composer.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "woocommerce/wc-api-dev",
+	"version": "v.0.8.4",
```

Comment:
This version does not follow the composer.json schema https://getcomposer.org/doc/04-schema.md#version because of the . between v and the major.